raitools/raitools/_managers/explainer_manager.py

```diff
@@ -117,7 +133,24 @@ def get(self):
             return []
 
     def list(self):
-        pass
+        """List information about the ExplainerManager.
+
+        :return: A dictionary of properties.
+        :rtype: dict
+        """
+        props = {ListProperties.MANAGER_TYPE: self.name}
+        if self._explanation:
+            props['id'] = self._explanation.id
```

Comment:
Should these keys be constants? And related, do we have a JSON schema for this?

Reply:
each manager defines its own dictionary of properties that it returns

Reply:
Doesn't mean that the keys shouldn't be constants. Or that each manager can't provide its own JSON Schema

Reply:
done, changed keys to constants